src/css/comment.css

```diff
@@ -0,0 +1,86 @@
+.comment-popup {
+  display: none;
+  background-color: hsl(216, 50%, 16%);
+  padding: 4px 15px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 98%;
+  height: 100vh;
+  z-index: 2;
+}
```

Comment:
- [ ] OPTIONAL:  When I open comments page, I cannot scroll down, this makes it impossible to click on comment button, I strongly suggest to fix this since this project can be a good showcase project on your portfolio 😉